### ISSUE TITLE
record founder of a monastery

### ISSUE THREAD

**Issue by eroux** [buda-base/owl-schema#39]
TL;DR:

maybe we could have a `:placeEventWho` property to record who is the responsible for a particular event. It would be mostly useful for the foundation of the monastery, for which we sometimes have this information.

Whole thing:

Looking at the Kagyu places out of curiosity, the first example I found is [G1KR1660](https://www.tbrc.org/#!rid=G1KR1660) which has a very long note indicating some interesting data, sadly in Tibetan only. It indicates that it was founded by འཇའ་ལུས་རྡོ་རྗེ shortly after 1900. This person has no record in the data, but there are many cases where it's would be easy to record it, for example [G4783](https://www.tbrc.org/#!rid=G4783) was founded by [P39](https://www.tbrc.org/#!rid=P39). Interestingly, there is a link between P39 and G4783 in the info of P39:

```xml
<p:event circa="11uu" type="foundsMonastery">
    <p:place pid="G4783">zhong zhong </p:place>
</p:event>
```

So this is a more convoluted case of symetric data... Basically here a personEvent and a placeEvent carry the exact same meaning, but one has less information than the other (the placeEvent doesn't record the person while the personEvent records the place). Maybe that could be modelled in a more uniform way?

**Comment by eroux** [comment]
Following https://github.com/BuddhistDigitalResourceCenter/owl-schema/issues/40 I think the problem is a bit larger here: even if we add `:placeEventWho`, we would have different notes and possibly different incoherent information... maybe I can make a pass in xmltold to detect incoherences and report them? I'm not sure what to do...

**Comment by eroux** [comment]
(BTW, in the previous examples, data contradict, we have in [P39](https://www.tbrc.org/xmldoc?rid=P39):

```xml
<p:event circa="11uu" type="foundsMonastery">
    <p:place pid="G4783">zhong zhong </p:place>
</p:event>
```

and in [G4783](https://www.tbrc.org/xmldoc?rid=G4783):

```xml
<g:event circa="1021" type="placeEventTypes:founded">
    <g:affiliation rid="lineage:ShangpaKagyu" type="placeEventAffiliationTypes:lineage">shangs pa bka' brgyud</g:affiliation>
</g:event>
```

, the date contradict (`11uu` vs `1021`)... I think it's typically the kind of annoyance that would be avoided if we have only one way to model something...

**Comment by eroux** [comment]
Another interesting though: even if we were to split the monastery foundation event between a person event and a monastery event, it would still be relevant to add the tradition affiliation, as people in one tradition don't always found monasteries of their tradition, on example would be the Shechen monastery [G20](https://www.tbrc.org/#!rid=G20), founded by [a Gelukpa](https://www.tbrc.org/xmldoc?rid=P661) as a Nyingma monastery, not a Gelugpa one

**Comment by xristy** [comment]
I'm reopening this until we have a solid solution to the :Event modeling
